pyke/keptrial.py

```diff
@@ -144,7 +145,8 @@ def keptrial(infile, outfile=None, datacol='SAP_FLUX', errcol='SAP_FLUX_ERR',
         # adjust data within the error bars
         #work1 = kepstat.randarray(signal, err)
         # determine FT power
-        fr, power = kepfourier.ft(barytime, signal, fmin, fmax, deltaf, False)
+        fr = np.arange(fmin,fmax,deltaf)
+        power = LombScargle(barytime, signal,signal.max()-signal.min()).power(fr)
```

Comment:
Nitpicking: let's always put a space behind a `,` as per PEP8, i.e. use `np.arange(fmin, fmax, deltaf)` instead of `np.arange(fmin,fmax,deltaf)`.

Reply:
Good point, updated those!